dts/arm/nordic/nrf52833.dtsi

```diff
@@ -367,6 +367,12 @@
 			status = "okay";
 			label = "WDT";
 		};
+
+		dfe_ant: dfe_ant {
```

Comment:
I will add it there also when run the code on particular hardware. Currently we support only nRF52833 and due to that I've added it to this dtsi.

Reply:
Now that the compatible has changed to `nordic,nrf-radio`, I think it should be a "proper" node corresponding to the RADIO peripheral, like so:

```
radio: radio@40001000 {
        compatible = "nordic,nrf-radio"
        reg = <0x40001000 0x1000>;
        interrupts = <1 NRF_DEFAULT_IRQ_PRIORITY>;
        status = "okay";
};
```

I would appreciate a double-check on those values from @cvinayak or another domain expert.

The status here is "okay" because I think we want to indicate that it is enabled by default. This is distinct from CONFIG_BT, which can still be set to `n` to disable software support for the radio. Otherwise it will be too much hassle to add overlays enabling the radio to every single BT sample.

Reply:
Controller has not been using DTS, but it is desired to move towards when time permits (contributions always welcome). I have no objections to having radio node.